.pre-commit-config.yaml

```diff
@@ -20,4 +21,3 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
-        args: [--settings-path=pyproject.toml]
```

Comment:
Since ts_pre_commit_conf is now used, this file may be deleted.

Reply:
Don't forget to update `pyproject.toml` as well and remove the configuration for `flake8` and `isort`.

Reply:
I will ask you to review this again to make sure I did it right.